utils/sort-by.js

```diff
@@ -1,13 +1,28 @@
+/*
+sort-by defaults to ascending order
+@param prop {String} The property to get from each item
```

Comment:
this should be `@prop {DataType} name` 
it won't currently break anything, but i imagine in the future when these params/props are actually used for warnings, etc. that it may